### PR TITLE
Update supported version of opal to 1.0

### PR DESCRIPTION
**Pull request by youchan**
It is going well on my own apps.
What are things that I must do to release updated version?

### CODE REVIEW

File: opal-haml.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.test_files     = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.require_paths  = ['lib']
 
-  s.add_dependency 'opal', ['~> 0.10', '< 0.12']
+  s.add_dependency 'opal', '~> 1.0'
```
Comment (elia) [inline opal-haml.gemspec]: Can you please replace `'~> 1.0'` with `['~> 0.10', '< 2']` so it will keep working with older Opal versions?

Comment (youchan) [reply opal-haml.gemspec]: 🆗 I have changed!

Comment (elia) [reply opal-haml.gemspec]: hmmm, sorry, actually I was wrong, based on this Travis failure the correct version requirement is `['>= 0.10', '< 2']`

https://travis-ci.org/opal/opal-haml/jobs/548229134

would you mind updating and checking if the build becomes green? 🙏

**Comment by elia** [comment]
[v0.4.6](https://rubygems.org/gems/opal-haml/versions/0.4.6) has been released! Thank you!

**Pull request merged**